.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: ['3.8']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install pip
+      run: pip install --upgrade pip
+    - name: Install Dependencies
+      run: pip install -r requirements/travis.txt
+    - name: Run Tests
+      run: tox -e py
+    - name: Run Quality Check
```

Comment:
This will run the Python tests, and then run the quality checks in the same worker.  For this repo it's probably fast enough, but I think we should get in the habit of running these in parallel in different `job`s.  Every little speedup in time to getting feedback makes it less likely that developers will need to context switch while waiting for CI results.